tests/test_datasets/test_dataset.py

```diff
@@ -320,6 +320,14 @@ def test_get_sparse_dataset_rowid_and_ignore_and_target(self):
         self.assertListEqual(categorical, [False] * 19998)
         self.assertEqual(y.shape, (600, ))
 
+    def test_get_sparse_categorical_data_id_395(self):
+        dataset = openml.datasets.get_dataset(395, download_data=False)
+        feature = dataset.features[3758]
+        self.assertEqual(dataset.name, 're1.wc')
+        self.assertEqual(feature.name, 'CLASS_LABEL')
+        self.assertEqual(feature.data_type, 'nominal')
+        self.assertEqual(len(feature.nominal_values), 25)
```

Comment:
Could you please add a check for the type of the output value?

Reply:
Added.

Reply:
Sorry for not being clear enough here. Could you please load X and y and check their type, dtype and shape?

Reply:
As discussed, I'm refraining from changing this test now. Have created an [issue](https://github.com/openml/openml-python/issues/827) to take care of such checks independently.